docs/reference/release-notes/highlights-7.3.0.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[release-highlights-7.3.0]]
+== 7.3.0 release highlights
+++++
+<titleabbrev>7.3.0</titleabbrev>
+++++
+
+coming[7.3.0]
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+[float]
+==== Voting-only master nodes
+
+A new <<voting-only-node,`node.voting-only`>> role has been introduced that
+allows nodes to participate in elections even though they are not eligible.
+The benefit is that these nodes still help with high availability while
+requiring less CPU and heap than master nodes.
+
+IMPORTANT: The `node.voting-only` role is only available with the default
+distribution of Elasticsearch.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Aliases are replicated via {ccr}
+
+Read aliases are now replicated via <<ccr-put-follow,{ccr}>>. Note that write
+aliases are still not replicated since they only make since for indices that
+are being written to while follower indices do not receive direct writes.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Memory leak when using templates in document-level security
```

Comment:
Maybe reword to make clear that the highlight is actually a fix for the potential memory leak? E.g.: "Limit memory usage when ..."?